modin/extension/extensions.py

```diff
@@ -0,0 +1,150 @@
+from typing import Any, Union
+
+import modin.pandas
```

Comment:
Does anyone have a strong opinion on this one?

Reply:
IIRC, this type of alert is raised when we import a module and something else from this module. I recall @anmyachev fixed some of such alerts some time ago. @anmyachev, do you remember the exact reason why this alert is raised?

Reply:
This is a kind of BKM that `CodeQL` recommends. Since we use them for the repo, I suggest following them here too. In this case, we can only use `import modin.pandas as pd` for everything.